plugin/evm/vm.go

```diff
@@ -816,7 +810,7 @@ func (vm *VM) initChainState(lastAcceptedBlock *types.Block) error {
 	}
 	vm.State = state
 
-	return vm.multiGatherer.Register(chainStateMetricsPrefix, chainStateRegisterer)
+	return vm.ctx.Metrics.Register(chainStateMetricsPrefix, chainStateRegisterer)
```

Comment:
I didn't seem to understand is part of the code correctly. I think this no long honors the `metrics.Enabled` flag.